randovania/games/cave_story/logic_database/Last Cave.txt

```diff
@@ -352,6 +347,27 @@ Extra - in_dark_aether: False
   > Exit to Passage?
       Trivial
 
+> After Spike Pit; Heals? False
+  * Layers: default
+  > H/V Trigger to Sacred Grounds - B2
+      All of the following:
+          Has Weapon
+          Any of the following:
+              Has Flight
+              # Damage boost off of spikes
+              Damage Boosting (Beginner) and Normal Damage ≥ 10
+  > Pickup (Hell B3 Chest)
```

Comment:
The chest goes directly to the H/V trigger. Should it not go trivially to `After Spike Pit`  instead?
It also has custom requirements to the heavy press that the other after spike pit node doesnt have.

Reply:
ah yeah I messed this one up, ty